CHANGELOG.rst

```diff
@@ -19,7 +27,20 @@ Changelog
     make sure to add ``besluittype.catalogus`` to the filters of the ``besluiten`` kanaal & ``informatieobjecttype.catalogus``
     to the filters of the ``documenten`` kanaal in ``notifications_kanalen_config``.
 
-* [:open-zaak:`1905`] Confirm support for Postgres versions 15 and 16 and Postgis 3.4 and 3.5
+**Bugfixes and QOL**
+
+* [:open-zaak:`1907`] Fix import mechanism incorrectly using sqlite as cache backend
```

Comment:
What do you think of backporting this issue and making patch release for OZ 1.18 ? 
I think it can be nice, but I'm not sure if it's really needed

Reply:
I think we discussed this during standup a week or so ago when you weren't there, but since we mostly do minor versions that do not really change the user experience (unlike Open Forms for example where minor versions can overhaul the UI), we decided to not do backports for these and instead just let people upgrade to the next minor version that includes the fix